src/components/backend-ai-usersettings-view.ts

```diff
@@ -236,16 +323,53 @@ export default class BackendAiUserSettingsView extends BackendAIPage {
   }
 
   async _editBootstrapScript() {
-    const editor = this.shadowRoot.querySelector('#codemirror-editor');
-    const dialog = this.shadowRoot.querySelector('#bootstrap-dialog');
     const script = await this._fetchBootstrapScript();
-    editor.setValue(script);
-    dialog.show();
+    this.bootstrapDialog.setValue(script);
```

Comment:
Why is it changed from `#codemirror-editor` to `#bootstrap-dialog`? I think bootstrap dialog does not have `setValue` method?